conanfile.py

```diff
@@ -75,6 +75,7 @@ class Morpheus(ConanFile):
         "gtest/1.13.0",
         "magic_enum/0.8.2",
         "ms-gsl/4.0.0",
+        "openssl/3.1.2", # to solve mold linker dependency conflict without force or override
```

Comment:
```suggestion
```

We should add this within the conditional check on line 105 so that we only override this when mold is in use.

Reply:
ok perfect!